wdl2cwl/main.py

```diff
@@ -33,9 +33,25 @@ def get_ram_min(ram_min: str) -> int:
 
     Only handles value given in GiB.
     """
-    unit = " ".join(re.findall("[a-zA-Z]+", ram_min))
+    unit = " ".join(re.findall("[a-zA-Z]+", ram_min)).replace(" ", "")
     ram_min = ram_min[ram_min.find('"') + 1 : ram_min.find(unit)]
-    return int(float(ram_min.strip()) * 1024)
+    ram_value = 0
+    # Add more units
+    if unit == "GiB":
+        ram_value = int(float(ram_min.strip()) * 1024)
+    return ram_value
+
+
+def get_ram_min_js(ram_min: str) -> str:
+    """Get memory requirement for user input."""
+    js_str = (
+        '${var unit = inputs["'
+        + ram_min
+        + '"].match(/[a-zA-Z]+/g).join("");\nvar value = parseInt(inputs["'
+        + ram_min
+        + '"].match(/[0-9]+/g));\nvar memory = "";\nif(unit==="GiB"){\n\tmemory = value*1024;\n}\nreturn memory;\n}'
```

Comment:
Great start! Now to implement the rest of the units: https://github.com/openwdl/wdl/blob/main/versions/1.1/SPEC.md#units-of-storage